src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -369,9 +370,7 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'c':
-                            sb.Append("{0:ddd} {0:MMM} ");
-                            sb.Append(StringUtil.Format("{0,2} ", dateTime.Day));
-                            sb.Append("{0:HH}:{0:mm}:{0:ss} {0:yyyy}");
+                            sb.Append("{0:ddd} {0:MMM} {0:dd} {0:HH}:{0:mm}:{0:ss} {0:yyyy}");
```

Comment:
This is different from Linux using `datetime -d "09/09/2017 08:30" +%c` which shows:

```none
Sat 09 Sept 2017 08:30:00 AM PDT
```

Not sure what the canonical representation should be

Reply:
I'll move a day number to second position.

Reply:
Fixed.